akka-docs/src/main/paradox/scala/stream/stream-io.md

```diff
@@ -110,6 +110,15 @@ of the server, and `q` in the case of the client. This is implemented by @scala[
 and concatenating a `Source` with a single `BYE` element which will then be sent after the original source completed]@java[using a custom `GraphStage`
 which completes the stream once it encounters such command].
 
+### Using bidi as framing protocol.
+
+Element boundary may get lost when streaming through Tcp connections.
+An end-of-frame marker, e.g. end line `\n`, can do framing via `Framing.delimiter`.
+Or a length-field can be used to build a framing protocol.
+There is a bidi implementing this protocol provided by `Framing.simpleFramingProtocol`,
+see api doc for more information.
```

Comment:
`simpleFramingProtocol` uses a length field, so should go together with that section or bullet if you turn it into a bullet list. 

Also: please link to the respective API-docs rather than just leaving it up to the reader to find them, http://doc.akka.io/api/akka/current/akka/stream/javadsl/Framing$.html#simpleFramingProtocol(maximumMessageLength:Int):akka.stream.javadsl.BidiFlow[akka.util.ByteString,akka.util.ByteString,akka.util.ByteString,akka.util.ByteString,akka.NotUsed] for Scala and http://doc.akka.io/japi/akka/current/akka/stream/javadsl/Framing.html#simpleFramingProtocol-int- for Java

You should be able to see above in the file how to include different content for Java and Scala